### ISSUE TITLE
Add shortcut to jump to first fv

### ISSUE THREAD

**Issue by blueyed** [awesomeWM/awesome#494]

**Comment by barlik** [comment]
I'm using modkey+Home to focus to master client. It works almost the same as yours code, but only on currently selected screen.
```
          -- Focus to master client
          awful.key({ modkey,           }, "Home",   function ()
                  c = awful.client.getmaster()
                  if c then
                          client.focus = c
                          c:raise()
                  end
          end),
```
I use it together with ```c:swap(awful.client.getmaster())```, which I've remapped from ``modkey+Ctrl+Return`` to ``modkey+Shift+Home``.

So I have ``modkey+Home`` jump to master client, and ``modkey+shift+Home`` to swap current client with master client.

I think that jumping to master client would make more sense to be included in the default config.

**Comment by Elv13** [comment]
@blueyed What's the status on this. Do you still think it should be added to the default config (as opposed to the doc)?

**Comment by blueyed** [comment]
I agree with @barlik here in general, and think those changes make sense.

Additionally, focusing / jumping to the first visible client also makes sense.

**Comment by blueyed** [comment]
My use case here is a tag filled up with 5+ clients, where I want to look at them and cleaning them up: I usually start with the left-most then.
